XCFramework/MUXSDKStats.xcframework/Info.plist

```diff
@@ -6,18 +6,16 @@
 	<array>
 		<dict>
 			<key>LibraryIdentifier</key>
-			<string>tvos-arm64_x86_64-simulator</string>
+			<string>ios-arm64_armv7</string>
```

Comment:
Is it ok to have this changes? 🤔

Reply:
tbh we always get them automatically haha and I'm not sure if there's a way to avoid them it might also be cause the xcode version I am using to generate the xcframework is different than the one in buildkite which I used to generate the last xcframework in the release branch. 

But yeah it doesn't hurt.